pontoon/contributors/static/js/profile.js

```diff
@@ -82,64 +84,68 @@ var Pontoon = (function (my) {
                 ),
                 pointHoverBorderColor: style.getPropertyValue('--white-1'),
                 order: 1,
+                fill: true,
+                tension: 0.4,
               },
             ],
           },
           options: {
-            legend: {
-              display: false,
-            },
-            tooltips: {
-              mode: 'index',
-              intersect: false,
-              borderColor: style.getPropertyValue('--status-translated'),
-              borderWidth: 1,
-              caretPadding: 5,
-              xPadding: 10,
-              yPadding: 10,
-              callbacks: {
-                label: function (items, chart) {
-                  const label = chart.datasets[items.datasetIndex].label;
-                  const value = nf.format(items.yLabel / 100);
-
-                  return `${label}: ${value}`;
+            clip: false,
+            plugins: {
+              tooltip: {
+                mode: 'index',
+                intersect: false,
+                borderColor: style.getPropertyValue('--status-translated'),
+                borderWidth: 1,
+                caretPadding: 5,
+                padding: {
+                  x: 10,
+                  y: 10,
+                },
+                callbacks: {
+                  labelColor: (context) => {
+                    return Pontoon.insights.setLabelColor(context);
+                  },
```

Comment:
You can [drop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#description) the body braces and the word "return":

```suggestion
                  labelColor: (context) =>
                    Pontoon.insights.setLabelColor(context),
                  },
```

The same applies for all other calls of this function.